olp-cpp-sdk-dataservice-read/src/repositories/DataRepository.h

```diff
@@ -29,10 +29,16 @@ namespace olp {
 namespace dataservice {
 namespace read {
 class DataRequest;
+class TileRequest;
```

Comment:
This doesn't work. Actually you can remove both fwd defines.
We already include Types.h and fwd defines only work for references or ptrs.

Reply:
There is no DataRequest in Types.h. Forward declarations changed to includes.